src/VuePhoneNumberInput/index.vue

```diff
@@ -114,22 +117,25 @@
       },
       locale () {
         const locale = this.defaultCountryCode || (!this.noUseBrowserLocale ? browserLocale() : null)
-        const countryAvailable = isCountryAvailable(locale)
+        const countryNameByLocale = countries.find(function(element) {
+          return element.iso2 === locale;
+        }).name;
```

Comment:
could run into null here. better to store the country on its own and check for null as an equivalent of isCountryAvailable(), then get the name after if it exists

Reply:
Thanks for pointing this out. Implement the latest changes